src/signals_notebook/entities/notebook.py

```diff
@@ -67,3 +69,24 @@ def create(
             force=force,
             request=request,
         )
+
+    def dump(self, base_path: str, fs_handler: FSHandler):
+        fs_handler.write(
+            fs_handler.join_path(base_path, self.eid, 'metadata.json'),
+            json.dumps({k: v for k, v in self.dict().items() if k in ('name', 'description', 'eid')}),
```

Comment:
You can just include or exclude fields as described [here](https://pydantic-docs.helpmanual.io/usage/exporting_models/#modeldict)
But, why don't you dump all the fields?